cmd/infracost/register.go

```diff
@@ -152,3 +169,20 @@ func promptOverwriteAPIKey() (bool, error) {
 
 	return true, nil
 }
+
+func promptGenerateNewKey() (bool, error) {
+	p := promptui.Prompt{
+		Label:     "Would you like to generate a new one?",
```

Comment:
```suggestion
		Label:     "Would you like to generate a new API key",
```

promptui adds a `?` by default

Reply:
will change it